tests/test_make.py

```diff
@@ -138,11 +138,13 @@ def test_make_stats(self) -> None:
         conf_path = Path("example_config.yaml")
         with open(conf_path, "r", encoding="utf8") as f:
             config = yaml.safe_load(f)
+        config_no_snsql = {"use-smartnoisesql": False, **config}
```

Comment:
Just on the off-chance that config already has a value for `use-smartnoisesql`, we could swap the order:

```suggestion
        config_no_snsql = {**config, "use-smartnoisesql": False}
```

Reply:
Done